proxysql/assets/dashboards/-_overview.json

```diff
@@ -0,0 +1,377 @@
+                                "alias": "Bytes sent"
+                            }
+                        },
+                        "q": "avg:proxysql.pool.bytes_data_sent{$proxysql_server,$proxysql_port} by {hostgroup}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Bytes received and sent per hostgroup",
+            "type": "timeseries",
+            "width": 47,
+            "x": 25,
+            "y": 42
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "14",
+            "height": 6,
+            "html": "# Performance",
+            "id": 7,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 47,
+            "x": 75,
+            "y": 2
+        },
+        {
+            "height": 13,
+            "id": 8,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "sum:proxysql.performance.command.cnt_5s{$proxysql_server,$proxysql_port,$sql_command} by {sql_command}.as_count()+sum:proxysql.performance.command.cnt_10s{$proxysql_server,$proxysql_port,$sql_command} by {sql_command}.as_count()+sum:proxysql.performance.command.cnt_infs{$proxysql_server,$proxysql_port,$sql_command} by {sql_command}.as_count()",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Number of statements that took more than 5 seconds to complete",
```

Comment:
You can probably add an alias to name metric names look nicer:
![image](https://user-images.githubusercontent.com/49917914/79570184-44cfde00-80b9-11ea-8116-14b463ef6e57.png)